scripts/debian/install-dev-dependencies.sh

```diff
@@ -63,18 +63,12 @@ apt-get -y --no-install-recommends install \
 
 codename="$(lsb_release --codename --short)"
 
-# Apache Arrow
+# Arrow ADBC
```

Comment:
Is this really just ADBC?

Reply:
That is correct, Arrow is now built from source.